webapp/package-lock.json

```diff
@@ -285,13 +285,11 @@
       }
     },
     "channels/node_modules/@mattermost/compass-components/node_modules/axios": {
-      "version": "1.7.9",
-      "resolved": "https://registry.npmjs.org/axios/-/axios-1.7.9.tgz",
-      "integrity": "sha512-LhLcE7Hbiryz8oMDdDptSrWowmB4Bl6RCt6sIJKpRB4XtVf0iEgewX3au/pJqm+Py1kCASkb/FFKjxQaLtxJvw==",
+      "version": "0.21.4",
```

Comment:
Can we revert these for this change? They probably should be updated but we'll do it in a separate change.